static/css/post.less

```diff
@@ -457,15 +457,24 @@
 
     .list_link {
         .h4();
-        .u-link__colors(@black, @pacific-80);
-        .u-link__no-border();
+        .u-link__colors(@pacific, @pacific, @pacific-50, @pacific, @navy);
+        border-width: 0;
+
+        &:hover {
+            border-bottom: 1px dotted;
```

Comment:
Is there any problem with not having the color defined here? I'm assuming the border is defined elsewhere and this is overriding the width and style?